rnd/autogpt_server/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 -m autogpt_server.app
```

Comment:
Use pyproject.toml scripts with poetry rather than requiring unix specific stuff. We gotta get away from non windows compatible stuff esp when using python, a very compatible language

https://stackoverflow.com/questions/59286983/how-to-run-a-script-using-pyproject-toml-settings-and-poetry

https://python-poetry.org/docs/pyproject/#scripts

Reply:
Moved this to pyproject.toml